Emby.Server.Implementations/Library/Resolvers/TV/SeasonResolver.cs

```diff
@@ -55,7 +55,7 @@ protected override Season Resolve(ItemResolveArgs args)
                     IndexNumber = seasonParserResult.SeasonNumber,
                     SeriesId = series.Id,
                     SeriesName = series.Name,
-                    Path = seasonParserResult.IsSeasonFolder ? path : args.Parent.Path
+                    Path = seasonParserResult.IsSeasonFolder ? path : null
```

Comment:
Quoting my Matrix message:

> Quite sure the LibraryManager sets the path anyway when it is null, so that if-statement might be useless now
> LibraryManager.ResolveItem uses the SeasonResolver, then uses ResolverHelper.SetInitialItemValues where it sets item.Path to args.Path

```suggestion
                    Path = path
```